tests/e2e/tests/test/e2e.ts

```diff
@@ -14,6 +14,7 @@ function _runServeAndE2e(...args: string[]) {
 
 export default function() {
   // This is supposed to fail without serving first...
+  console.log(bold(yellow('This test is expected to throw Error, Don\'t worry.')));
```

Comment:
I'm afraid not. See [travis log here](https://travis-ci.org/angular/angular-cli/jobs/186520222#L3818-L3868)